composer.json

```diff
@@ -40,7 +40,13 @@
     "bin": ["bin/jackalope", "bin/jackrabbit.sh"],
     "extra": {
         "branch-alias": {
-            "dev-master": "1.2-dev"
+            "dev-master": "1.3-dev"
         }
+    },
+  "repositories": [
+    {
+      "type": "vcs",
+      "url":  "https://github.com/markussr/jackalope.git"
```

Comment:
i just merged https://github.com/jackalope/jackalope/pull/302, can you please remove the repository redirection again? i think then this PR is ready to merge as well. please squash the commits and then i can merge it